it_IT.md

```diff
@@ -54,37 +54,36 @@ Il Design è una soluzione intenzionale ad un problema ottenuta operando all'int
+Quando decidi per chi stai progettando, stai facendo una dichiarazione implicita su chi non stai progettando. Per anni abbiamo chiamato “casi limite” le persone che non erano determinanti per il successo dei nostri prodotti. Stavamo emarginando le persone. E stavamo prendendo la decisione che c'erano persone nel mondo i cui problemi non valevano la pena di essere risolti.
 
-Facebook now claims to have two billion users. 1% of two billion people, which most products would consider an edge case, is twenty million people. Those are the people at the margins.
+Facebook ora afferma di avere due miliardi di utenti. L'1% di due miliardi di persone, che la maggior parte dei prodotti considererebbe un caso limite, è di venti milioni di persone. Quelle sono le persone ai margini.
 
-“When you call something an edge case, you’re really just defining the limits of what you care about.” — Eric Meyer
-These are the trans people who get caught on the edges of “real names” projects. These are the single moms who get caught on the edges of “both parents must sign” permission slips. These are the elderly immigrants who show up to vote and can’t get ballots in their native tongues.
+"Quando chiami qualcosa un caso limite, stai solo definendo i limiti di ciò a cui tieni". — Eric Meyer. Queste sono le persone trans che rimangono intrappolate ai margini dei progetti con "nomi veri". Queste sono le mamme single che rimangono intrappolate ai margini delle schede di autorizzazione "entrambi i genitori devono firmare". Questi sono gli immigrati anziani che si presentano per votare e non possono ottenere schede nelle loro lingue native.
 
-They are not edge cases. They are human beings, and we owe them our best work.
+Non sono casi limite. Sono esseri umani e dobbiamo loro il nostro miglior lavoro.
 
-### A designer is part of a professional community.
+### Un designer fa parte di una comunità professionale.
 
-You are part of a professional community and the way you do your job and handle yourself professionally affects everyone in that community. Just as a rising tide affects all boats, taking a shit in the pool affects all swimmers. If you are dishonest with a client or employer, the designer behind you will pay the price. If you work for free, the designer behind you will be expected to do the same. If you do not hold your ground on doing bad work, the designer behind you will have to work twice as hard to make up for it.
+Fai parte di una comunità professionale e il modo in cui svolgi il tuo lavoro e ti comporti professionalmente influisce su tutti in quella comunità. Proprio come una marea crescente colpisce tutte le barche, cagare in piscina colpisce tutti i nuotatori. Se sei disonesto con un cliente o un datore di lavoro, il designer dietro di te ne pagherà il prezzo. Se lavori gratis, il designer dietro di te dovrà fare lo stesso. Se non mantieni la tua posizione nel fare un cattivo lavoro, il designer dietro di te dovrà lavorare il doppio per rimediare.
```

Comment:
Fai parte di una comunità professionale e il modo in cui svolgi il tuo lavoro e ti comporti professionalmente influisce su tutti in quella comunità. Proprio come una marea crescente colpisce tutte le barche, cagare in piscina riguarda tutti i nuotatori. Se sei disonesto con un cliente o un datore di lavoro, il designer dietro di te ne pagherà il prezzo. Se lavori gratis, il designer dietro di te dovrà fare lo stesso. Se non mantieni la tua posizione nel fare un cattivo lavoro, il designer dietro di te dovrà lavorare il doppio per rimediare.

Reply:
I wish there were a way to avoid the article in this passage, too. _Un'altro designer_ still sounds masculine. I'll integrate these fixes. Thanks!

Reply:
I would ask a content designer to make a further review, but in Italian, if I was using "un designer dietro di te" sounded very weird. Moreover even though "Un designer" is indefinite masculine, in Italian we do have also "una designer" as feminine, so is not perfectly achieving the scope. I should have used un designer e una designer to achieve an inclusive language.

Reply:
It's so hard with romance languages. French is my native tongue, and it's virtually impossible to do. I appreciate your work!